tests/tfhe_rust_bool/end_to_end_fpga/test_cggi_add_bool.mlir

```diff
@@ -0,0 +1,77 @@
+// This test ensures the testing harness is working properly with minimal codegen.
+
+// RUN: heir-opt --straight-line-vectorize --cggi-to-tfhe-rust-bool -cse -remove-dead-values %s | heir-translate --emit-tfhe-rust-bool > %S/src/fn_under_test.rs
+// RUN: cargo run --release --manifest-path %S/Cargo.toml -- 1 1 | FileCheck %s
+
+#encoding = #lwe.unspecified_bit_field_encoding<cleartext_bitwidth = 1>
+!ct_ty = !lwe.lwe_ciphertext<encoding = #encoding>
+!pt_ty = !lwe.lwe_plaintext<encoding = #encoding>
+
+// CHECK: 01000000
+func.func @fn_under_test(%arg0: tensor<8x!ct_ty>, %arg1: tensor<8x!ct_ty>) -> tensor<8x!ct_ty> {
```

Comment:
Maybe add a comment indicating what this function is supposed to do? It would help to understand how to fix it if something causes the test to fail in the future.